### DISCUSSION TITLE
Ender 3 V2 DWIN icon set

### DISCUSSION THREAD

**Discussion by pbarwell** [mriscoc/Ender3V2S1#1130] [Q&A] [answered]
How do I make an icon set for my Ender 3 V2 DWIN display?
I would also like to ask for the resolution and file types the images for the boot screen would be.
Additionally, I have no idea of how to flash the new display firmware, or how to compile the printer firmware ].

**Comment by mriscoc** [comment, marked as answer]
https://github.com/mriscoc/Ender3V2S1/tree/Ender3V2S1-Released/buildroot/share/dwin/bin
